Document for command `gp timeout` (#2218)
gitpod-io/website: gitpod/docs/command-line-interface.md

  sync-await          Awaits an event triggered using gp sync-done
  sync-done           Notifies the corresponding gp sync-await calls that this event has happened
  tasks               Interact with workspace tasks

>   timeout             A collection of commands for workspace timeout

Reviewer: **Thought:** Maybe this should be plural, e.g. "timeouts" to match the ports/tasks convention... however, a timeout is also a singular item 🤔 💭

Author: We can add other commands like `reset` for timeout, but define `timeout` as a command only will make alias etc in the future not works

Author: I can't update cmd description above because it should align to current `GP` command 😦, I will file a fast PR to update cmd description into below

> Interact with a workspaces `timeout` configuration.

gp tasks attach <id>
```

## timeout

> A commands collection of workspace `timeout`

Reviewer: **Suggestion:** Update wording. 

```suggestion
Interact with a workspaces `timeout` configuration.
```

Author: Can we go with below? I'm not sure why need a `a` before workspace.

> Commands to interact with `workspace timeout` configuration.

### extend

> Extend workspace timeout for current workspace, which will make other workspaces' timeout reset to default.

Reviewer: **Question:** Possibly I'm not aware of this behavioror, but what does: "which will make other workspaces' timeout reset to default", mean?

Author: actually it's a limit, we only allow one workspace extend timeout at same time

Reviewer 2: What about this?

```suggestion
Extends the current workspace's timeout. 

> **Note:** You can only have one workspace with extended timeout at a time. This means executing this command with another timeout-extended workspace will result in that workspace's timeout to be reset.
```

Author: It'll `which will make other workspaces' timeout reset to default.` that is needed, otherwise, user may confuse about why those workspaces timed out

Author: `**Note:** You can only have one workspace with extended timeout at a time.` is enough?

> Default timeout and extended timeout depend on your [plans](https://gitpod.io/plans) or [team plans](https://gitpod.io/teams)

Reviewer: ```suggestion
The default timeout, and the ability to extend a workspace timeout depends on your [plan](https://gitpod.io/plans) or [team plans](https://gitpod.io/teams)
```

gp tasks attach <id>
```

## timeout

Interact with workspace timeout configuration.

### extend

Extend workspace timeout for current workspace, which will make other workspaces' timeout reset to default.

> The default timeout, and the ability to extend a workspace timeout depends on your [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams)

Reviewer 2: We can point to the existing docs 👀 

```suggestion
The default timeout, and the ability to extend a workspace timeout depends on your [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams). You can learn more in [Life of a Workspace](https://www.gitpod.io/docs/life-of-workspace#timeouts). 
```
